stdlib/FileWatching/src/FileWatching.jl

```diff
@@ -649,10 +649,13 @@ giving the result of the polling.
 function poll_fd(s::Union{RawFD, Sys.iswindows() ? WindowsRawSocket : Union{}}, timeout_s::Real=-1; readable=false, writable=false)
     wt = Condition()
     fdw = _FDWatcher(s, readable, writable)
+    local timer
```

Comment:
Just curious, but why not `timer = nothing` and later `timer === nothing || close(timer)`?  Is it some kind of optimization or just a coding style?

Reply:
True, those would be essentially same. So just equivalent variations based on how I started writing it.